Assignment2/pyproject.toml

```diff
@@ -10,7 +10,8 @@ authors = [
 dependencies = [
 "matplotlib",
     "numpy",
-
+    "pytest",
```

Comment:
this should be a dev dependency, as it is only for developers will not be exported for "production"

Reply:
Done